code/modules/shuttle/custom_shuttle/shuttle_creator_console.dm

```diff
@@ -0,0 +1,95 @@
+//============The internal camera console used for designating the area=============
+/obj/machinery/computer/camera_advanced/shuttle_creator
+	name = "internal shuttle creator console"
+	desc = "You should not have access to this, please report this as a bug"
+	networks = list()
+	var/obj/item/shuttle_creator/owner_rsd
+	var/datum/action/innate/shuttle_creator/designate_area/area_action = new
+	var/datum/action/innate/shuttle_creator/designate_turf/turf_action = new
+	var/datum/action/innate/shuttle_creator/clear_turf/clear_turf_action = new
+	var/datum/action/innate/shuttle_creator/reset/reset_action = new
+	var/datum/action/innate/shuttle_creator/airlock/airlock_action = new
```

Comment:
are these referenced managed properly?
also do your setup in Initialize not in the typedef